### PR TITLE
refactor(crux): url prefix to image name prefix

### PR DESCRIPTION
**Pull request by peterambrus3**
refactor(crux): url prefix to image name prefix
chore: run prettier format

### CODE REVIEW

File: web/crux/prisma/migrations/20220815104826_url_prefix_to_image_name_prefix/migration.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
+-- Copy data
+DROP TABLE IF EXISTS public._migration_instance;
+SELECT 
+	id,
+  "Registry"."urlPrefix"
+INTO public._migration_instance
```
Comment (robot9706) [inline web/crux/prisma/migrations/20220815104826_url_prefix_to_image_name_prefix/migration.sql:6]: `_migration_registry` would be a better name

**Review by balanceee — APPROVED** [review]
LGTM

**Review by robot9706 — CHANGES_REQUESTED** [review]

File: web/crux/prisma/migrations/20220815104826_url_prefix_to_image_name_prefix/migration.sql (hunk 1 above)
Comment (peterambrus3) [reply web/crux/prisma/migrations/20220815104826_url_prefix_to_image_name_prefix/migration.sql:6]: true, my bad sry

File: web/crux-ui/src/const.ts [hunk 2]
```diff
@@ -25,4 +25,3 @@ export const ATTRIB_CSRF = 'csrf_token'
 export const HEADER_SET_COOKIE = 'set-cookie'
 export const AUTH_RESEND_DELAY = 30 // seconds
 export const WEBOOK_TEST_DELAY = 500 //millis
-
```
Comment (polaroi8d) [inline web/crux-ui/src/const.ts:28]: If we remove there will be no NW at EOF, not?

**Review by polaroi8d — APPROVED** [review]
LGTM

**Review by robot9706 — APPROVED** [review]
😎

**Pull request merged**